File: C/Matriz/Cada Linha.c
#include <stdio.h>
int main(){


int i,j,n, maior;

    printf("Qual a ordem da matriz? ");
    scanf("%d", &n);
        int mat[n][n], vet[n];

            for(i=0;i<n;i++){
                for(j=0;j<n;j++){
                    printf("Elemento [%d,%d]:", i,j);
                    scanf("%d", &mat[i][j]);
                }
            }


                for(i=0;i<n;i++){
                        maior = mat[i][0];
                    for(j=0;j<n;j++){
                     if (mat[i][j] > maior){
                        maior = mat[i][j];
                     }
                }
                vet[i]=maior;
            }
 printf("MAIOR ELEMENTO DE CADA LINHA:\n");
                for(i=0; i<n; i++){
                    printf("%d\n", vet[i]);
                }
}

File: Phyton/Matriz/cadaLinha.py
n = int(input("Qual a ordem da matriz ?"))

mat = [[0 for x in range(n)]for x in range(n)]

for i in range(n):
    for j in range(n):
        mat[i][j] = int(input(f"Elemento [{i},{j}]:"))
print("MAIOR ELEMENTO DE CADA LINHA: ")
for i in range(n):
    maior = mat[i][0]
    for j in range(n):
        if mat[i][j] > maior:
            maior = mat[i][j]
    print(maior)



